{{ cookiecutter.repo_name }}/deploy/docker-compose.yml

```diff
@@ -1,68 +1,27 @@
 version: '2'
 services:
-    worker:
-        image: drtools/{{cookiecutter.repo_name}}-worker:dev
-        build:
-          context: .
-          dockerfile: dev.dockerfile
-
     luigid:
-        image: drtools/{{cookiecutter.repo_name}}-worker:dev
```

Comment:
Should we rely on latest image, or stick to one version and maybe update it manually periodically? Not suggesting, just asking.